esp/src/eclwatch/DynamicESDLDetailsWidget.js

```diff
@@ -260,7 +267,7 @@ define([
             var currSel = this.getSelectedChild();
             if (currSel.id == this.widget._Summary.id && !this.widget._Summary.__hpcc_initalized) {
                 this.widget._Summary.__hpcc_initalized = true;
-                var table = domConstruct.create("table", {});
+                var table = domConstruct.create("table", {id: this.id + "_serviceInformation"});
```

Comment:
Its probably ok here - but the "_" will get interpreted as being a tab for the "#" id - probably best to remove it.